src/main/java/wolox/reactortraining/services/BotService.java

```diff
@@ -129,17 +167,17 @@ private static Topic createTopicModel(TopicDto dto, User user) {
             .collectList();
     }
 
-    private Mono<Void> createBot(String botName, List<String> topics) {
+    private Mono<Void> feedBot(String botName, List<String> topics) {
         return twitterService
             .getTweetsStreamPipe()
             .map(Tweet::getText)
             .take(500)
```

Comment:
put in constants or environment var

Reply:
Lo arreglaré, en develop ya estaba pero no se porque no paso el cambio